### DISCUSSION TITLE
ClickHouse  JDBC：datetime类型无法解析

### DISCUSSION THREAD

**Discussion by rulerhuang** [apache/doris#26049] [A - General / Q&A] [answered]
【系统环境】
doris 2.0.2
clickhouse 22.12.6.12

【复现步骤】
1、在Doris中创建 ClickHouse JDBC
```hiveql
CREATE CATALOG jdbc_clickhouse PROPERTIES (
    "type"="jdbc",
    "user"="xx",
    "password"="xx",
    "jdbc_url" = "jdbc:clickhouse://127.0.0.1:8123/dev_lake",
    "driver_url" = "clickhouse-jdbc-0.4.2.jar",
    "driver_class" = "com.clickhouse.jdbc.ClickHouseDriver"
);
``` 
2、切换catalog，查询外表中数据
```hiveql
select count(*) from defect_changelogs;
``` 
引发如下异常：
![image](https://github.com/apache/doris/assets/14310763/0526bcbd-50be-4bd3-a136-c83db5213b36)

3、Clickhouse 中defect_changelogs表结构如下：
```hiveql
CREATE TABLE defect_changlogs
(
    `issue_id` UInt64 COMMENT 'issue_id',
    `created_at` DateTime('Asia/Shanghai') COMMENT '记录创建时间',
    -- 省略其他字段
)
ENGINE = ReplacingMergeTree()
``` 

4、改造defect_changelogs，去掉DateTime类型的字段，重新创建catalog，可以顺利执行select count(*)
![image](https://github.com/apache/doris/assets/14310763/02fc4322-4925-4ebf-a26f-66a2e410882b)

【结论】
Doris的ClickHouse JDBC，对DateTime类型的字段不支持，其他类型的字段没有问题。

PS: clickhouse-jdbc-0.4.2.jar 换成较新的 clickhouse-jdbc-0.4.6.jar 也是一样的错误

**Comment by liugddx** [comment, marked as answer]
I will deal with this problem
